### PR TITLE
feat: New Crowdin updates

### PR DESCRIPTION
**Pull request by teolemon**

### CODE REVIEW

**Review by stephanegigandet — APPROVED** [review]
I looked a bit at the diffs in en and fr, but not every change.

File: po/openbeautyfacts/sl.po [hunk 1]
```diff
@@ -29,7 +29,7 @@ msgstr ""
 
 msgctxt "site_name"
 msgid "Open Beauty Facts"
-msgstr "Open Beauty Facts"
+msgstr "Odpri dejstva lepote"
```
Comment (teolemon) [inline po/openbeautyfacts/sl.po:32]:
```suggestion
msgstr "Open Beauty Facts"
```

File: po/openpetfoodfacts/nl_NL.po [hunk 2]
```diff
@@ -92,7 +92,7 @@ msgstr "EMB 53062"
 
 msgctxt "site_description"
 msgid "A collaborative, free and open database of ingredients, nutrition facts and information on pet food products from around the world"
-msgstr "Een gezamenlijke, vrije en open database met ingrediënten, voedingsfeiten en informatie over dierenvoedsel uit de hele wereld"
+msgstr "Een collaboratieve, gratis en open database met ingrediënten, voedingsfeiten en informatie over dierenvoedsel uit de hele wereld"
```
Comment (teolemon) [inline po/openpetfoodfacts/nl_NL.po:95]:
```suggestion
msgstr "Een gezamenlijke, vrije en open database met ingrediënten, voedingsfeiten en informatie over dierenvoedsel uit de hele wereld"
```

File: po/openpetfoodfacts/openpetfoodfacts.pot [hunk 3]
```diff
@@ -60,8 +61,8 @@ msgid "https://apps.apple.com/app/open-beauty-facts/id1122926380"
 msgstr ""
 
 msgctxt "windows_phone_app_link"
-msgid "https://apps.microsoft.com/store/detail/open-food-facts-scan-to-get-nutriscore-ecoscore-and-more/XP8LT18SRPKLRG"
-msgstr "https://apps.microsoft.com/store/detail/open-food-facts-scan-to-get-nutriscore-ecoscore-and-more/XP8LT18SRPKLRG"
+msgid "https://www.microsoft.com/p/openfoodfacts/9nblggh0dkqr"
+msgstr ""
```
Comment (teolemon) [inline po/openpetfoodfacts/openpetfoodfacts.pot:65]:
```suggestion
msgid "https://www.microsoft.com/p/openfoodfacts/9nblggh0dkqr"
msgstr ""
```
```suggestion
msgid "https://apps.microsoft.com/store/detail/open-food-facts-scan-to-get-nutriscore-ecoscore-and-more/XP8LT18SRPKLRG"
msgstr "https://apps.microsoft.com/store/detail/open-food-facts-scan-to-get-nutriscore-ecoscore-and-more/XP8LT18SRPKLRG"
```

File: po/common/fr.po [hunk 4]
```diff
@@ -2919,12 +2903,12 @@ msgid "Risk of overexposure"
...
+msgid "The European Food Safety Authority (EFSA) has determined that some population groups have a high risk of reaching or exceeding the Acceptable Daily Intake (ADI) for <tag>."
+msgstr "L'Autorité européenne de sécurité des aliments (EFSA) a déterminé que certains groupes de population ont un risque élevé d'atteindre ou de dépasser la Dose Journalière Admissible (DJA) pour <tag>."
 
 msgctxt "additives_efsa_evaluation_overexposure_risk_moderate"
-msgid "The European Food Safety Authority (EFSA) has determined that some population groups have a moderate risk of consuming too much <tag>."
-msgstr "L'Autorité Européenne de Sécurité des Aliments (AESA/EFSA) a déterminé que certains groupes de population ont un risque modéré de consommer trop de <tag>."
+msgid "The European Food Safety Authority (EFSA) has determined that some population groups have a moderate risk of reaching or exceeding the Acceptable Daily Intake (ADI) for <tag>."
+msgstr "L'Autorité européenne de sécurité des aliments (EFSA) a déterminé que certains groupes de population ont un risque modéré d'atteindre ou de dépasser la Dose Journalière Admissible (DJA) pour <tag>."
```
Comment (teolemon) [inline po/common/fr.po:2911]:
```suggestion
msgstr "L'Autorité Européenne de Sécurité des Aliments (AESA/EFSA) a déterminé que certains groupes de population ont un risque modéré de consommer trop de <tag>."
```

File: html/donate/pt_PT.html [hunk 5]
```diff
@@ -211,24 +243,29 @@ <h3>Help us to fund the Open Food Facts 2022 budget!</h3>
...
-              <strong> Pessoas e projetos: </strong> Dar o nosso melhor apoio à comunidade da Open
-              Food Facts, trabalhar com pesquisadores e fazer com que os nossos dados de
-              alimento tenham o maior impacto na nossa saúde, o nosso planeta
-              e na sociedade.
-            </p>
+              <strong>People and projects:</strong> To give our best support to the Open
+              Food Facts community, to work with researchers, and to make our
+              food products data have the biggest impact on our health, our
+              planet, and society.
+            </p -->
```
Comment (teolemon) [inline html/donate/pt_PT.html:263]:
```suggestion
            </p> -->
```

File: html/donate/zh_CN.html [hunk 6]
```diff
@@ -211,24 +245,29 @@ <h3>Help us to fund the Open Food Facts 2022 budget!</h3>
...
-              <strong>人员和项目：</strong>
-              Food Facts 社区通过与研究人员合作，提供最好的支持。对我们的
-              食品数据对我们的健康、我们的
-              地球和社会产生最大的影响。
-            </p>
+              <strong>People and projects:</strong> To give our best support to the Open
+              Food Facts community, to work with researchers, and to make our
+              food products data have the biggest impact on our health, our
+              planet, and society.
+            </p -->
```
Comment (teolemon) [inline html/donate/zh_CN.html:265]:
```suggestion
            </p> -->
```

File: html/donate/zh_CN.html [hunk 7]
```diff
@@ -211,24 +245,29 @@ <h3>Help us to fund the Open Food Facts 2022 budget!</h3>
             width="100%"
           ></iframe>
 
-          <div class="text-left">
-            <h3>您的捐款去向</h3>
+          <div class="text-center">
+            <img src="https://static.openfoodfacts.org/images/misc/2023-donation/join-the-food-rev.png"
+                 alt="加入食品革命！" style="margin-top: -15%; margin-bottom: -20%; width:80%"
+            >
+            <!-- h3>Where your donation goes</h3>
```
Comment (teolemon) [inline html/donate/zh_CN.html:252]:
```suggestion
            <!-- <h3>Where your donation goes</h3>
```

File: html/donate/pt_PT.html [hunk 8]
```diff
@@ -211,24 +243,29 @@ <h3>Help us to fund the Open Food Facts 2022 budget!</h3>
             width="100%"
           ></iframe>
 
-          <div class="text-left">
-            <h3>Para onde vai sua doação</h3>
+          <div class="text-center">
+            <img src="https://static.openfoodfacts.org/images/misc/2023-donation/join-the-food-rev.png"
+                 alt="Junta-te à revolução alimentar!" style="margin-top: -15%; margin-bottom: -20%; width:80%"
+            >
+            <!-- h3>Where your donation goes</h3>
```
Comment (teolemon) [inline html/donate/pt_PT.html:250]:
```suggestion
            <!-- <h3>Where your donation goes</h3>
```

File: po/common/fr.po [hunk 9]
```diff
@@ -2919,11 +2903,11 @@ msgid "Risk of overexposure"
 msgstr "Risque de sur-exposition"
 
 msgctxt "additives_efsa_evaluation_overexposure_risk_high"
-msgid "The European Food Safety Authority (EFSA) has determined that some population groups have a high risk of consuming too much <tag>."
-msgstr "L'Autorité Européenne de Sécurité des Aliments (AESA/EFSA) a déterminé que certains groupes de population ont un risque élevé de consommer trop de <tag>."
+msgid "The European Food Safety Authority (EFSA) has determined that some population groups have a high risk of reaching or exceeding the Acceptable Daily Intake (ADI) for <tag>."
+msgstr "L'Autorité européenne de sécurité des aliments (EFSA) a déterminé que certains groupes de population ont un risque élevé d'atteindre ou de dépasser la Dose Journalière Admissible (DJA) pour <tag>."
```
Comment (teolemon) [inline po/common/fr.po:2907]:
```suggestion
msgstr "L'Autorité Européenne de Sécurité des Aliments (AESA/EFSA) a déterminé que certains groupes de population ont un risque élevé de consommer trop de <tag>."
```

File: po/common/fr.po [hunk 10]
```diff
@@ -2951,12 +2935,12 @@ msgid "No or very low risk of over exposure"
...
-msgid "Risk of exceeding the acceptable daily intake (ADI)"
-msgstr "Risque de dépasser la dose journalière admissible (DJA)"
+msgid "Risk of reaching or exceeding the acceptable daily intake (ADI)"
+msgstr "Risque d'atteindre ou de dépasser la dose journalière admissible (DJA)"
 
 msgctxt "additives_efsa_evaluation_exposure_greater_than_noael"
-msgid "Risk of exceeding the maximum dose without adverse effect (No observed adverse effect level - NOAEL)"
-msgstr "Risque de dépasser la dose maximale sans effet nocif observable (DSENO)"
+msgid "Risk of reaching exceeding the maximum dose without adverse effect (No observed adverse effect level - NOAEL)"
+msgstr "Risque d'atteindre ou dépasser la dose maximale sans effet indésirable (dose sans effet nocif observable - DSENO)"
```
Comment (teolemon) [inline po/common/fr.po:2943]:
```suggestion
msgstr "Risque de dépasser la dose maximale sans effet nocif observable (DSENO)"
```

File: po/common/fr.po [hunk 11]
```diff
@@ -2975,20 +2959,20 @@ msgid "Groups with more than 5% of members exceeding the maximum dose without ad
...
-msgid "Most people"
-msgstr "La plupart des gens"
+msgid "High risk"
+msgstr "Risque élevé"
 
 msgctxt "exposure_description_mean"
-msgid "over 50%"
-msgstr "plus de 50%"
+msgid "50% of people or more"
+msgstr "50% des personnes ou plus"
```
Comment (teolemon) [inline po/common/fr.po:2975]:
```suggestion
msgid "over 50%"
msgstr "plus de 50%"
```

File: po/common/fr.po [hunk 12]
```diff
@@ -2975,20 +2959,20 @@ msgid "Groups with more than 5% of members exceeding the maximum dose without ad
...
-msgid "over 5%"
-msgstr "plus de 5%"
+msgid "5% of people or more"
+msgstr "5% des personnes ou plus"
 
 msgctxt "exposure_title_mean"
-msgid "Most people"
-msgstr "La plupart des gens"
+msgid "High risk"
+msgstr "Risque élevé"
```
Comment (teolemon) [inline po/common/fr.po:2971]:
```suggestion
msgid "Most people"
msgstr "La plupart des gens"
```

File: po/common/fr.po [hunk 13]
```diff
@@ -2975,20 +2959,20 @@ msgid "Groups with more than 5% of members exceeding the maximum dose without ad
...
-msgid "Some people"
-msgstr "Certaines personnes"
+msgid "Moderate risk"
+msgstr "Risque modéré"
 
 msgctxt "exposure_description_95th"
-msgid "over 5%"
-msgstr "plus de 5%"
+msgid "5% of people or more"
+msgstr "5% des personnes ou plus"
```
Comment (teolemon) [inline po/common/fr.po:2967]:
```suggestion
msgid "over 5%"
msgstr "plus de 5%"
```

File: po/common/fr.po [hunk 14]
```diff
@@ -2975,20 +2959,20 @@ msgid "Groups with more than 5% of members exceeding the maximum dose without ad
 msgstr "Groupes avec plus de 5% de membres qui dépassent la dose maximale sans effet nocif observable (DSENO)"
 
 msgctxt "exposure_title_95th"
-msgid "Some people"
-msgstr "Certaines personnes"
+msgid "Moderate risk"
+msgstr "Risque modéré"
```
Comment (teolemon) [inline po/common/fr.po:2963]:
```suggestion
msgid "Some people"
msgstr "Certaines personnes"
```

File: po/common/fr.po [hunk 15]
```diff
@@ -2919,11 +2903,11 @@ msgid "Risk of overexposure"
...
-msgstr "L'Autorité Européenne de Sécurité des Aliments (AESA/EFSA) a déterminé que certains groupes de population ont un risque élevé de consommer trop de <tag>."
+msgid "The European Food Safety Authority (EFSA) has determined that some population groups have a high risk of reaching or exceeding the Acceptable Daily Intake (ADI) for <tag>."
+msgstr "L'Autorité européenne de sécurité des aliments (EFSA) a déterminé que certains groupes de population ont un risque élevé d'atteindre ou de dépasser la Dose Journalière Admissible (DJA) pour <tag>."
 
 msgctxt "additives_efsa_evaluation_overexposure_risk_moderate"
-msgid "The European Food Safety Authority (EFSA) has determined that some population groups have a moderate risk of consuming too much <tag>."
+msgid "The European Food Safety Authority (EFSA) has determined that some population groups have a moderate risk of reaching or exceeding the Acceptable Daily Intake (ADI) for <tag>."
 msgstr "L'Autorité Européenne de Sécurité des Aliments (AESA/EFSA) a déterminé que certains groupes de population ont un risque modéré de consommer trop de <tag>."
```
Comment (teolemon) [inline po/common/fr.po:2911]:
```suggestion
msgstr "L'Autorité européenne de sécurité des aliments (EFSA) a déterminé que certains groupes de population ont un risque modéré d'atteindre ou de dépasser la Dose Journalière Admissible (DJA) pour <tag>."
```

**Comment by teolemon** [comment]
Structures begin differing at:
#          $got->{data_quality_warnings}{title_element}{subtitle} = 'We have detected some potential issues or potential improvements, could you check if some apply ?'
#     $expected->{data_quality_warnings}{title_element}{subtitle} = 'We have detected some potential issues or potential improvement, could you check if some apply ?'
#

File: po/openpetfoodfacts/nl_NL.po (hunk 2 above)
Comment (teolemon) [inline po/openpetfoodfacts/nl_NL.po:95]:
```suggestion
msgstr "Een gezamenlijke, vrije en open database met ingrediënten, voedingsfeiten en informatie over dierenvoedsel uit de hele wereld"
```

File: po/openbeautyfacts/ar.po [hunk 16]
```diff
@@ -29,7 +29,7 @@ msgstr "https://apps.apple.com/app/open-beauty-facts/id1122926380"
 
 msgctxt "site_name"
 msgid "Open Beauty Facts"
-msgstr "Open Beauty Facts"
+msgstr "حقائق الجمال مفتوحة"
```
Comment (teolemon) [inline po/openbeautyfacts/ar.po:32]:
```suggestion
msgstr "Open Beauty Facts"
```

File: po/openbeautyfacts/ne.po [hunk 17]
```diff
@@ -126,7 +126,7 @@ msgstr ""
 # Do not translate
 msgctxt "footer_obf"
 msgid "Open Food Facts"
-msgstr "Open Food Facts"
+msgstr ""
```
Comment (teolemon) [inline po/openbeautyfacts/ne.po:129]:
```suggestion
msgstr "Open Food Facts"
```

File: po/openbeautyfacts/uk.po [hunk 18]
```diff
@@ -126,7 +126,7 @@ msgstr "/open-beauty-facts-mobile-app"
 # Do not translate
 msgctxt "footer_obf"
 msgid "Open Food Facts"
-msgstr "Open Food Facts"
+msgstr ""
```
Comment (teolemon) [inline po/openbeautyfacts/uk.po:129]:
```suggestion
msgstr "Open Food Facts"
```

File: po/common/pt_BR.po [hunk 19]
```diff
@@ -2360,7 +2360,7 @@ msgstr "Um banco de dados colaborativo, livre e aberto, com ingredientes, inform
 
 msgctxt "site_name"
 msgid "Open Food Facts"
-msgstr "Open Food Facts"
+msgstr ""
```
Comment (teolemon) [inline po/common/pt_BR.po:2363]:
```suggestion
msgstr "Open Food Facts"
```

File: po/common/ne.po [hunk 20]
```diff
@@ -2344,7 +2344,7 @@ msgstr ""
 
 msgctxt "site_name"
 msgid "Open Food Facts"
-msgstr "Open Food Facts"
+msgstr ""
```
Comment (teolemon) [inline po/common/ne.po:2347]:
```suggestion
msgstr "Open Food Facts"
```

File: po/common/uz.po [hunk 21]
```diff
@@ -2345,7 +2345,7 @@ msgstr ""
 
 msgctxt "site_name"
 msgid "Open Food Facts"
-msgstr "Open Food Facts"
+msgstr ""
```
Comment (teolemon) [inline po/common/uz.po:2348]:
```suggestion
msgstr "Open Food Facts"
```

File: po/openpetfoodfacts/nl_BE.po [hunk 22]
```diff
@@ -92,7 +92,7 @@ msgstr "EMB 53062"
 
 msgctxt "site_description"
 msgid "A collaborative, free and open database of ingredients, nutrition facts and information on pet food products from around the world"
-msgstr "Een gezamenlijke, vrije en open database met ingrediënten, voedingsfeiten en informatie over dierenvoedsel uit de hele wereld"
+msgstr "Een collaboratieve, gratis en open database met ingrediënten, voedingsfeiten en informatie over dierenvoedsel uit de hele wereld"
```
Comment (teolemon) [inline po/openpetfoodfacts/nl_BE.po:95]:
```suggestion
msgstr "Een gezamenlijke, vrije en open database met ingrediënten, voedingsfeiten en informatie over dierenvoedsel uit de hele wereld"
```

**Comment by teolemon** [comment]
#   Failed test 'openbeautyfacts: 'bottom_content' in 'sv' should contain '<<site_name>>''
#   at tests/unit/i18n.t line 25.
#                   '<a href="https://world.openfoodfacts.org/donate-to-open-food-facts">Donate to Open Beauty Facts</a><p><<site_name> �r skapad av en ideell f�rening, oberoende av industrin. Den �r gjord f�r allt, av alla och �r finansierad av alla. Du kan st�dja v�rt arbete genom att <a href="https://world.openfoodfacts.org/donate-to-open-food-facts">donera till Open Beauty Facts</a> och genom att <a href="https://www.lilo.org/fr/open-food-facts/?utm_source=open-food-facts">anv�nda Lilo-s�kmotorn</a>.<br/><b>Tack!</b></p>'
#     doesn't match '(?^u:<<site_name>>)'

#   Failed test 'openbeautyfacts: 'bottom_content' in 'uk' should contain '<<site_name>>''
#   at tests/unit/i18n.t line 25.
Wide character in print at /opt/perl/local/lib/perl5/Test2/Formatter/TAP.pm line 125.
#                   '<a href="https://world.openfoodfacts.org/donate-to-open-food-facts">Donate to Open Beauty Facts</a><p><site_name> створено некомерційною асоціацією, незалежною від галузі. Вона створена для всіх, всіма, і фінансується всіма. Ви можете підтримати наші старання пожертвою на <a href="https://world.openfoodfacts.org/donate-to-open-food-facts">для Open Beauty Facts</a> а також на <a href="https://www.lilo.org/fr/open-food-facts/?utm_source=open-food-facts">, використовуючи пошукову систему Lilo</a>.<br/><b>Дякуємо вам!</b></p>'
#     doesn't match '(?^u:<<site_name>>)'

#   Failed test 'openpetfoodfacts: 'openfoodhunt_points' in 'hu' should contain '<<site_name>>''
#   at tests/unit/i18n.t line 25.
Wide character in print at /opt/perl/local/lib/perl5/Test2/Formatter/TAP.pm line 125.
#                   'Február 21. szombattól március 1. vasárnapig <a href="/open-beauty-hunt">Nyitott állateledel vadászat</a> a <<site_name> oldalon! A Közreműködők díjazásban részesülnek
# Felfedezés pontokat kapnak az általuk hozzáadott termékekért és Nagykövet pontokat az általuk toborzott új közreműködőkért. A pontok 30 percenként frissülnek.'
#     doesn't match '(?^u:<<site_name>>)'

#   Failed test 'openpetfoodfacts: 'openfoodhunt_points' in 'fr' should contain '<<site_name>>''
#   at tests/unit/i18n.t line 25.
#                   'C'est <a href="/open-beauty-hunt">Open Product Hunt</a> sur <site_name> du samedi 21 f�vrier au dimanche 1er mars !
# Les contributeurs sont r�compens�s via des Points Explorateur pour les produits qu'ils ajoutent et des Points Ambassadeur pour les nouveaux contributeurs qu'ils recrutent. Le nombre de points est actualis� toutes les 30 minutes.'
#     doesn't match '(?^u:<<site_name>>)'

#   Failed test 'openpetfoodfacts: 'bottom_content' in 'sv' should contain '<<site_name>>''
#   at tests/unit/i18n.t line 25.
#                   '<a href="https://world.openfoodfacts.org/donate-to-open-food-facts">Donate to Open Beauty Facts</a><p><<site_name> �r skapad av en ideell f�rening, oberoende av industrin. Den �r gjord f�r allt, av alla och �r finansierad av alla. Du kan st�dja v�rt arbete genom att <a href="https://world.openfoodfacts.org/donate-to-open-food-facts">donera till Open Beauty Facts</a> och genom att <a href="https://www.lilo.org/fr/open-food-facts/?utm_source=open-food-facts">anv�nda Lilo-s�kmotorn</a>.<br/><b>Tack!</b></p>'
#     doesn't match '(?^u:<<site_name>>)'

#   Failed test 'openpetfoodfacts: 'bottom_content' in 'uk' should contain '<<site_name>>''
#   at tests/unit/i18n.t line 25.
Wide character in print at /opt/perl/local/lib/perl5/Test2/Formatter/TAP.pm line 125.
#                   '<a href="https://world.openfoodfacts.org/donate-to-open-food-facts">Donate to Open Beauty Facts</a><p><site_name> створено некомерційною асоціацією, незалежною від галузі. Вона створена для всіх, всіма, і фінансується всіма. Ви можете підтримати наші старання пожертвою на <a href="https://world.openfoodfacts.org/donate-to-open-food-facts">для Open Beauty Facts</a> а також на <a href="https://www.lilo.org/fr/open-food-facts/?utm_source=open-food-facts">, використовуючи пошукову систему Lilo</a>.<br/><b>Дякуємо вам!</b></p>'
#     doesn't match '(?^u:<<site_name>>)'

#   Failed test 'openproductsfacts: 'openfoodhunt_points' in 'ko' should contain '<<site_name>>''
#   at tests/unit/i18n.t line 25.
Wide character in print at /opt/perl/local/lib/perl5/Test2/Formatter/TAP.pm line 125.
#                   '2월 21 일 토요일부터 3 월 1 일 일요일까지 <site_name>에서 < <site_name> >의 <a href="/open-product-hunt">Open Product Hunt</a> 입니다! 기여자에게는 추가 한 제품에 대해 탐험가 점수와, 새로 모집 한 신규 기여자에 대한 대사 점수가 부여됩니다. 포인트는 30 분마다 업데이트됩니다.'
#     doesn't match '(?^u:<<site_name>>)'

#   Failed test 'openproductsfacts: 'openfoodhunt_points' in 'fr' should contain '<<site_name>>''
#   at tests/unit/i18n.t line 25.
#                   'C'est <a href="/open-product-hunt">Open Product Hunt</a> sur <site_name> du samedi 21 f�vrier au dimanche 1er mars !
# Les contributeurs sont r�compens�s via des Points Explorateur pour les produits qu'ils ajoutent et des Points Ambassadeur pour les nouveaux contributeurs qu'ils recrutent. Le nombre de points est actualis� toutes les 30 minutes.'
#     doesn't match '(?^u:<<site_name>>)'

#   Failed test 'openproductsfacts: 'bottom_content' in 'pt' should contain '<<site_name>>''
#   at tests/unit/i18n.t line 25.
#                   '<a href="https://world-pt.openfoodfacts.org/fazer-um-donativo-ao-open-food-facts">Fazer um donativo ao Open Products Facts</a><p><site_name>> � um projeto gerido por uma associa��o sem fins lucrativos, independente da ind�stria. � feito para todos, por todos e � financiado por todos. Pode apoiar o nosso trabalho <a href="https://world-pt.openfoodfacts.org/fazer-um-donativo-ao-open-food-facts">fazendo um donativo ao Open Products Facts</a> e tamb�m <a href="https://www.lilo.org/fr/open-food-facts/?utm_source=open-food-facts">utilizando o motor de busca Lilo</a>.<br/><b>Obrigado!</b></p>'
#     doesn't match '(?^u:<<site_name>>)'

#   Failed test 'openproductsfacts: 'bottom_content' in 'it' should contain '<<site_name>>''
#   at tests/unit/i18n.t line 25.
#                   'Donate to Open Products Facts � fatto da un'associazione non-profit, indipendente dall'industria. Fatta per tutti, da tutti, e finanziata da tutti. Puoi supportare il nostro lavoro <a href="https://world.openfoodfacts.org/donate-to-open-food-facts"><a href="https://world.openfoodfacts.org/donate-to-open-food-facts">donando a Open Products Facts</a> ed anche <a href="https://www.lilo.org/fr/open-food-facts/?utm_source=open-food-facts">usando il motore di ricerca Lilo</a>.<br/><b>Grazie!</b></p>'
#     doesn't match '(?^u:<<site_name>>)'
# Looks like you failed 10 tests of 1768.
tests/unit/i18n.t ............................. 
Dubious, test returned 10 (wstat 2560, 0xa00)
Failed 10/1768 subtests

**Comment by teolemon** [comment]
@stephanegigandet I've attempted correcting 4 tests, but it throws more errors,  and this time I don't understand what's going on (eg in the Korean open pet food facts file)

**Comment by teolemon** [comment]
I already got that, but it hasn't been changed for what I see.

**Comment by teolemon** [comment]
```
#   Failed test at tests/unit/display.t line 132.
#                   '<!-- start templates/web/pages/product/includes/nutriscore_details.tt.html -->
# 
# <p>This product is not considered a beverage for the calculation of the Nutri-Score.</p>
# 
# 
# 
# 
# 
#   <p> 
#     <strong>Positive points: 2</strong> 
#   </p>
# 
#   <ul>
#     
#       <li>
#         <strong>Proteins:
# 2&nbsp;</strong>/&nbsp;5
# (value: 3.9, 
#         rounded value: 3.9)
#       </li>
#     
#       <li>
#         <strong>Fiber:
# 0&nbsp;</strong>/&nbsp;5
# (value: 0, 
#         rounded value: 0)
#       </li>
#     
#       <li>
#         <strong>Fruits, vegetables, nuts, and colza/walnut/olive oils:
# 0&nbsp;</strong>/&nbsp;5
# (value: 0, 
#         rounded value: 0)
#       </li>
#     
#   </ul>
# 
# 
# 
#   <p> 
#     <strong>Negative points: 8</strong> 
#   </p>
# 
#   <ul>
#     
#       <li>
#         <strong>Energy:
# 1&nbsp;</strong>/&nbsp;10
# (value: 573, 
#         rounded value: 573)
#       </li>
#     
#       <li>
#         <strong>Sugars:
# 3&nbsp;</strong>/&nbsp;10
# (value: 15, 
#         rounded value: 15)
#       </li>
#     
#       <li>
#         <strong>Saturated fat:
# 3&nbsp;</strong>/&nbsp;10
# (value: 3.5, 
#         rounded value: 3.5)
#       </li>
#     
#       <li>
#         <strong>Sodium:
# 1&nbsp;</strong>/&nbsp;10
# (value: 160, 
#         rounded value: 160)
#       </li>
#     
#   </ul>
# 
# 
# 
# <p>The points for proteins are counted because the negative points are less than 11.</p>
# 
# <p>
#   <strong>Nutritional score: 6</strong>
#   (8 - 2)
# </p>
# <p><strong>Nutri-Score: C</strong></p>
# 
# <!-- end templates/web/pages/product/includes/nutriscore_details.tt.html -->'
#     doesn't match '(?^u:Score nutritionnel: 6)'
Use of uninitialized value $ProductOpener::Display::formatted_subdomain in concatenation (.) or string at /opt/product-opener/lib/ProductOpener/Display.pm line 3132.
```

File: po/common/fr.po [hunk 23]
```diff
@@ -4776,6 +4759,10 @@ msgctxt "attribute_ecoscore_setting_name"
 msgid "Low environmental impact (Eco-Score)"
 msgstr "Faible impact environnemental (Eco-Score)"
 
+msgctxt "attribute_ecoscore_setting_note"
+msgid "The Eco-Score is an environmental score (ecoscore) from A to E which makes it easy to compare the impact of food products on the environment."
+msgstr "Eco-Score: L'Éco-Score est un score environnemental (écoscore) de A à E qui permet de comparer facilement l'impact des produits alimentaires sur l'environnement."
```
Comment (teolemon) [inline po/common/fr.po:4764]:
```suggestion
msgstr "L'Éco-Score est un score environnemental (écoscore) de A à E qui permet de comparer facilement l'impact des produits alimentaires sur l'environnement."
```

**Pull request merged**